cfgmgr/nbrmgr.cpp

```diff
@@ -257,3 +284,20 @@ void NbrMgr::doTask(Consumer &consumer)
         it = consumer.m_toSync.erase(it);
     }
 }
+
+void NbrMgr::doTask(Consumer &consumer)
+{
+    string table_name = consumer.getTableName();
+
+    if (table_name == CFG_NEIGH_TABLE_NAME)
+    {
+        doSetNeighTask(consumer);
+    }
+
+    if (table_name == APP_NEIGH_RESOLVE_TABLE_NAME)
+    {
+        doResolveNeighTask(consumer);
+    }
+
+    SWSS_LOG_ERROR("Unknown REDIS table %s ", table_name.c_str());
```

Comment:
Looks like missed in if-else. Please remove or place in if-else

Reply:
Thanks, corrected!